script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -957,7 +991,7 @@ Eighteen hours later, I finally caught you."
 [sha:df5bbaa21cd86d7ae776f6a20b18197508e1760e]{
 -- Page 156, Offset 6001.
 -- かつん、と更に一歩、女は路地裏に入ってくる。
-With a clunk, the woman takes another step into the alleyway.
+I hear a tap as the woman takes another step into the alleyway.
```

Comment:
"tap" in italics

Reply:
I don't think that's in line with what we've been doing - onomatope only get italics treatment if they're voiced non-words or standalone fragments iirc

Reply:
it might necessarily be consistent with what we've been doing but it would make this line look a lot nicer